jina/clients/base/http.py

```diff
@@ -77,6 +77,13 @@ def _result_handler(result):
                     r_str = await response.json()
                     if r_status == 404:
                         raise BadClient(f'no such endpoint {url}')
+                    elif (
+                        r_status == 503
```

Comment:
is it possible to have this code and not have the rest of the info? I would handle it at 2 different levels

Reply:
I guess, but not sure if that is so useful. If we want to show the `description` in the header to the user, then we should check that it is there, otherwise some different (and not understandable) exception will be thrown.
So if we don't have the additional checks then we can't do anything useful, so that case would be the same as the one below with `elif r_status < 200 or r_status > 300:`.